assets/js/ModuleLibrary.js

```diff
@@ -118,12 +118,16 @@ export default class ModuleLibrary extends React.Component {
   toggleLibrary() {
     if (!this.props.isReadOnly) {
       this.setState({ libraryOpen: !this.state.libraryOpen });
+      // make API call to set
```

Comment:
A rare instance where I'd say the comment isn't helping much. General guideline: comment the why, not the what.